package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "angular2",
+    "name": "Angular2",
```

Comment:
The name/id of this extension in the vscode marketplace is johnpapa.Angular2 and I think the package.json 'name' has to match the marketplace .  The proposed fix does work when I manually changed the name on the installed package.json.